packages/frontend/src/hooks/useQueryResults.tsx

```diff
@@ -135,7 +154,7 @@ export const useSavedChartResults = (
     return useQuery<ApiQueryResults, ApiError>({
         queryKey,
         queryFn: () =>
-            getQueryResults({
+            getDashboardTileQueryResults({
```

Comment:
Is the `useSavedChartResults` hook only used on dashboard tiles?

Reply:
Yep 
![image](https://user-images.githubusercontent.com/1983672/227172969-96eb2e92-33bc-4dec-a7e7-82f8c4283123.png)